plugins/compiler.js

```diff
@@ -53,8 +53,11 @@
   update node start/end positions of other nodes
   we're interested in */
   function replaceNode(node, newCode) {
+    // TODO: Where is 'code' defined? Is this a global?
     code = code.substr(0,node.start) + newCode + code.substr(node.end);
     var offs = newCode.length - (node.end-node.start); // offset for future code snippets
+
+    // TODO: Also unclear where 'tasks' is defined, there's nothing in this scope to declare it
     for (var i in tasks)
```

Comment:
There are a few definitions of `tasks`, this one doesn't _appear_ to be defined to the linter, so maybe it's a global?

Reply:
I think as for `code` above - it was intended that this function be defined in `compileCode`